addons/point_of_sale/point_of_sale_view.xml

```diff
@@ -36,6 +36,7 @@
                                     <field name="qty" on_change="onchange_qty(product_id, discount, qty, price_unit, context)"/>
                                     <field name="price_unit" on_change="onchange_qty(product_id, discount, qty, price_unit, context)"/>
                                     <field name="discount"  on_change="onchange_qty(product_id, discount, qty, price_unit, context)"/>
+                                    <field name="tax_ids"/>
```

Comment:
Better use widget="many2many_tags" in case someone wants to edit the taxes - pretty unusable otherwise ;-)